Azure/sonic-buildimage
# [bgp]: Fix isolate/unisolate command for ipv6 peers

**- What I did**
Fixed bug https://github.com/Azure/sonic-buildimage/issues/3070

**- How I did it**
I introduced `address-family ipv6` for ipv6 peers

**- How to verify it**
Build the image, run it on your dut, use `bgp-isolate` command and after that check that output of `show ipv6 bgp neighbor <ipv6 peer> adv`. You should see all aspathes are increased.

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
